package.json

```diff
@@ -69,6 +69,7 @@
     "url": "http://github.com/localForage/localForage/issues"
   },
   "dependencies": {
+    "jslint": "^0.12.1",
```

Comment:
This should be in `devDependencies`, not `dependencies`, as you don't need `jslint` to run the repo.

That said: this shouldn't need to be here at all. If we're using a linter it should be `eslint`, but we aren't using a linter (or, specifically, `jslint`) anywhere in the codebase or this PR. Could you remove this line please?

Reply:
Ok

Reply:
Sorry, I forget that this type of dependency came under dev dependencies.